magefile.go

```diff
@@ -1423,7 +1409,49 @@ func (Integration) TestOnRemote(ctx context.Context) error {
 	return nil
 }
 
-func createTestRunner(batches ...define.Batch) (*runner.Runner, error) {
+func integRunner(ctx context.Context, matrix bool, singleTest string) error {
+	batches, err := define.DetermineBatches("testing/integration", "integration")
+	if err != nil {
+		return fmt.Errorf("failed to detemine batches: %w", err)
+	}
+	r, err := createTestRunner(matrix, singleTest, batches...)
+	if err != nil {
+		return err
+	}
+	results, err := r.Run(ctx)
+	if err != nil {
+		return err
+	}
+	_ = os.Remove("build/TEST-go-integration.out")
+	_ = os.Remove("build/TEST-go-integration.out.json")
+	_ = os.Remove("build/TEST-go-integration.xml")
+	err = writeFile("build/TEST-go-integration.out", results.Output, 0644)
+	if err != nil {
+		return err
+	}
+	err = writeFile("build/TEST-go-integration.out.json", results.Output, 0644)
+	if err != nil {
+		return err
+	}
+	err = writeFile("build/TEST-go-integration.xml", results.XMLOutput, 0644)
+	if err != nil {
+		return err
+	}
+	if results.Failures > 0 {
+		fmt.Printf(">>> Testing completed (%d failures)\n", results.Failures)
+	} else {
+		fmt.Printf(">>> Testing completed (%d successful)\n", results.Tests)
+	}
```

Comment:
[Suggestion]
Print both, the success and failures for the failures case.
```suggestion
	if results.Failures > 0 {
		fmt.Printf(">>> Testing completed (%d failures, %d successful)\n",
		 results.Failures, results.Tests-results.Failures)
	} else {
		fmt.Printf(">>> Testing completed (%d successful)\n", results.Tests)
	}
```